Assets/MixedRealityToolkit/_Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -60,11 +64,12 @@ public enum DeviceInputType
         Start,
         Menu,
         Hand,
-        Thumb,
```

Comment:
Thumb should stay. It's a separate input from these other's you added.

Reply:
Thumb is still there, it's just been expanded to include Press, Touch and NearTouch

Reply:
I don't see "Thumb" by itself.

Reply:
Fair enough, I'll add "Thumb" alone back in.